adserver/models.py

```diff
@@ -802,6 +814,16 @@ class AdType(TimeStampedModel, models.Model):
         ),
     )
 
+    publisher_groups = models.ManyToManyField(
+        PublisherGroup,
+        related_name="adtypes",
+        blank=True,
```

Comment:
m2m's default to null=True right? (eg. No objects)

Reply:
Yes